patches/tModLoader/Terraria/ModLoader/UI/UIModSources.cs

```diff
@@ -225,6 +230,57 @@ internal class UIModSources : UIState, IHaveBackButtonCommand
 			modListViewPosition = _modList.ViewPosition;
 		}
 
+		private bool ShouldSkipPopulate() {
```

Comment:
Names like `Should` imply a lack of side-effects. Consider instead `ShowInfoMessages`